frontend/packages/dev-console/src/components/pipelines/PipelineAugmentRuns.tsx

```diff
@@ -1,29 +1,60 @@
 import * as React from 'react';
-import { Table } from '@console/internal/components/factory';
-import PipelineHeader from './PipelineHeader';
-import PipelineRow from './PipelineRow';
+import { inject } from '@console/internal/components/utils';
+import { K8sKind } from '@console/internal/module/k8s';
 import { augmentRunsToData, PropPipelineData, KeyedRuns } from '../../utils/pipeline-augment';
-import { PipelineModel } from '../../models';
+import { pipelineFilterReducer, pipelineStatusFilter } from '../../utils/pipeline-filter-reducer';
+
+interface ListPipelineData extends K8sKind {
+  data: PropPipelineData[];
+}
+export const filters = [
+  {
+    type: 'pipeline-status',
+    selected: ['Running', 'Failed', 'Succeeded'],
+    reducer: pipelineFilterReducer,
+    items: [
+      { id: 'Running', title: 'Running' },
+      { id: 'Failed', title: 'Failed' },
+      { id: 'Succeeded', title: 'Succeeded' },
+    ],
+    filter: pipelineStatusFilter,
+  },
+];
 
 export type PipelineAugmentRunsProps = {
   data?: PropPipelineData[];
   propsReferenceForRuns?: string[];
+  pipeline?: ListPipelineData;
+  reduxIDs?: string[];
+  applyFilter?: () => void;
+  filters?: Record<string, any>[];
 };
-
 // Firehose injects a lot of props and some of those are considered the KeyedRuns
 const PipelineAugmentRuns: React.FC<PipelineAugmentRunsProps> = ({
-  data,
+  pipeline,
   propsReferenceForRuns,
   ...props
-}) => (
-  <Table
-    {...props}
-    data={augmentRunsToData(data, propsReferenceForRuns, props as KeyedRuns)}
-    aria-label={PipelineModel.labelPlural}
-    Header={PipelineHeader}
-    Row={PipelineRow}
-    virtualize
-  />
-);
+}) => {
+  const paramFilters = {};
+  const params = new URLSearchParams(window.location.search);
+  params.forEach((v, k) => {
+    paramFilters[k] = v;
+  });
+
+  const resourceData =
+    pipeline && pipeline.data && propsReferenceForRuns
+      ? augmentRunsToData(pipeline.data, propsReferenceForRuns, props as KeyedRuns)
+      : null;
+
+  const children = inject(props.children, {
+    resources: { pipeline: { data: resourceData } },
+    pipeline,
+    reduxIDs: props.reduxIDs,
+    applyFilter: props.applyFilter,
+    loaded: true,
+    filters: { ...props.filters, ...paramFilters },
+  });
+  return <>{children}</>;
```

Comment:
nit: we use it verbosely everywhere else in the code. makes it more readable in my opinion.
```suggestion
  return <React.Fragment>{children}</Reacct.Fragment>;
```